src/main/java/com/kainos/ea/dao/SpecificationsDao.java

```diff
@@ -0,0 +1,43 @@
+package com.kainos.ea.dao;
+
+
+import com.kainos.ea.exception.EmptyListException;
+import com.kainos.ea.model.JobSpecification;
+
+import javax.ws.rs.core.Response;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+
+public class SpecificationsDao {
```

Comment:
you can reuse the role dao as you are getting info from the roles table

Reply:
done